nearlib/dev.js

```diff
@@ -29,15 +30,17 @@ module.exports = {
             fullRuntimeOptions.accountId = devAccountName;
             fullRuntimeOptions.key = devKey;
         }
+        fullRuntimeOptions.networkId = fullRuntimeOptions.networkId || 'localhost';
         fullRuntimeOptions.nodeUrl = fullRuntimeOptions.nodeUrl || (await this.getConfig()).nodeUrl || localNodeUrl;
-        fullRuntimeOptions.deps.keyStore = fullRuntimeOptions.deps.keyStore || new nearlib.BrowserLocalStorageKeystore(),
+        fullRuntimeOptions.deps.keyStore = fullRuntimeOptions.deps.keyStore || new nearlib.BrowserLocalStorageKeystore(fullRuntimeOptions.networkId),
         fullRuntimeOptions.deps.storage = fullRuntimeOptions.deps.storage || window.localStorage;
         this.deps = fullRuntimeOptions.deps;
+        this.options = fullRuntimeOptions;
         const nearClient = new nearlib.NearClient(
             new nearlib.SimpleKeyStoreSigner(this.deps.keyStore), new nearlib.LocalNodeConnection(fullRuntimeOptions.nodeUrl));
         this.near = new nearlib.Near(nearClient);
         if (fullRuntimeOptions.accountId && fullRuntimeOptions.key) {
-            this.deps.keyStore.setKey(fullRuntimeOptions.accountId, fullRuntimeOptions.key);
+            this.deps.keyStore.setKey(fullRuntimeOptions.accountId, fullRuntimeOptions.key, fullRuntimeOptions.networkId);
```

Comment:
Hmm, so this is passing `networkId` to `setKey`, but looks like `setKey` doesn't expect it (uses `this.networkId`).
 
I like approach of key store knowing it's `networkId`, passing `networkId` each time seems like a hassle.

Reply:
fixed. Old code when I looked how hard it would be to pass as param